srcpkgs/mod_md/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'mod_md'
+pkgname=mod_md
+version=2.4.21
+revision=1
+build_style=gnu-configure
+configure_args="--with-apxs=/usr/bin/apxs --enable-werror"
```

Comment:
we shouldn't be enabling Werror, compiler updates often add new warnings which would break the build for no real reason

Reply:
fixed